Author: Adam Sutton <dev@adamsutton.me.uk>
Date:   Sun Jul 29 20:37:00 2012 +0100

    Fix false trigger of uninit var, detected using gcc v4.7.1 (compiling openelec tvh add-on), reported by seo.

diff --git a/src/htsp.c b/src/htsp.c
index b9fc3e1..ae5bd51 100644
--- a/src/htsp.c
+++ b/src/htsp.c
@@ -1160,7 +1160,7 @@ htsp_read_message(htsp_connection_t *htsp, htsmsg_t **mp, int timeout)
 static int
 htsp_read_loop(htsp_connection_t *htsp)
 {
-  htsmsg_t *m, *reply;
+  htsmsg_t *m = NULL, *reply;
   int r, i;
   const char *method;

